tests/Unit/Controllers/Api/UsersControllerTest.php

```diff
@@ -117,4 +119,73 @@ public function testUserById(): void
         $this->assertEquals($otherUser->id, $data['data']['id']);
         $this->assertArrayNotHasKey('dates', $data['data']);
     }
+
+    /**
+     * @covers \Engelsystem\Controllers\Api\UsersController::entriesByAngeltype
+     * @covers \Engelsystem\Controllers\Api\Resources\UserAngelTypeReferenceResource::toArray
+     */
+    public function testEntriesByAngeltype(): void
+    {
+        /** @var User $user */
+        $user = User::factory()->create();
+        $controller = new UsersController(new Response());
+        /** @var AngelType $angelType */
+        $angelType = AngelType::factory()->create(['restricted' => false]);
```

Comment:
Why not test with a restricted angel type?

Reply:
That was part of a , now not needed, overwrite -> removed